modules/unsupported/ysld/src/main/java/org/geotools/ysld/Band.java

```diff
@@ -0,0 +1,92 @@
+/*
+ *    GeoTools - The Open Source Java GIS Toolkit
+ *    http://geotools.org
+ *
+ *    (C) 2014-2016, Open Source Geospatial Foundation (OSGeo)
+ *    
```

Comment:
This is one for @smithkm - should we have a (C) 2015-2016 Boundless to reflect this code has shipped in another product?